components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -94,12 +94,39 @@
   #set ($collaborators = $request.getParameterValues('collaborator'))
   #set ($accessLevels = $request.getParameterValues('accessLevel'))
   #set ($owner = "$!{request.get('owner')}")
+  #if ("$!{owner}" != "")
```

Comment:
You can change the collaborators with an empty owner. Note that the owner in this case is the *new owner* that this patient is being transferred to. An empty $owner variable refers to no change in ownership.